CONTRIBUTING.md

```diff
@@ -23,12 +23,10 @@ Thanks for your interest in contributing to Ionic's documentation! :tada: Check
 In order to run the documentation locally, install the dependencies and run the development server:
 
 ```sh
-$ npm install --legacy-peer-deps
-$ npm start
+npm install
+npm start
```

Comment:
The margins seem larger now:

(top is branch, bottom is main. I added red guidelines for emphasis)

![image](https://github.com/ionic-team/ionic-docs/assets/2721089/747ebbda-972f-4398-85e5-cef929723f73)

Also looks like the default arrow icon changed too.

Reply:
If you do see some differences. It's due to Docusaurus changing how they calculate the columns.